### PR TITLE
Add PPP prittam

### PR DESCRIPTION
**Pull request by prit3010**
Updated Add PPP Prittam

### CODE REVIEW

File: docs/team/prit3010.md [hunk 1]
```diff
@@ -9,4 +9,63 @@ title: Prittam Ravi's Project Portfolio Page
...
+---
+
+#### Code Contributed
+
+#### Enhancements Implemented
+* **New Features**
+  * Implemented `batch-add` feature
+    * Created a `BatchAddCommand` class to handle the logic (Refer to code [here](https://github.com/AY2223S1-CS2103T-T15-1/tp/blob/master/src/main/java/coydir/logic/commands/BatchAddCommand.java)) 
+    and `BatchAddCommandParser` class to handle the parsing of the user input (Refer to code [here](https://github.com/AY2223S1-CS2103T-T15-1/tp/blob/master/src/main/java/coydir/logic/parser/BatchAddCommandParser.java))
+    * Use Case: User can add multiple employees at once if the user has them inputted in a `.csv` file
```
Comment (Kok-je) [inline docs/team/prit3010.md:23]: abit of grammar issue here with the "if the user has them inputted ..." 

Consier "User can add multiple employees at once if they have the data in the form of a `.csv` file" or something to that extent.

File: docs/team/prit3010.md [hunk 2]
```diff
@@ -9,4 +9,63 @@ title: Prittam Ravi's Project Portfolio Page
...
+  * Interface Layout
+
+#### Contributions to the DG
+* Added implementation details for:
+  * Batch-Add function
+  * [To Be Updated]
+
+#### Contributions to the team-based tasks
+* Helped to create and distribute issues 
+* Did heavy manual testing of the application to find for bugs
```
Comment (Kok-je) [inline docs/team/prit3010.md:60]: either "find bugs" or "look for bugs", slight grammar issue

**Review by Kok-je — CHANGES_REQUESTED** [review]
Overall LGTM, detailed PPP other than some minor phrasing/ grammar errors

File: docs/team/prit3010.md [hunk 3]
```diff
@@ -9,4 +9,63 @@ title: Prittam Ravi's Project Portfolio Page
...
+    One functionality of this feature is that if there is any error, the command will be aborted 
+    and the database will be reverted to previous state. This was a bit more tricky as the functions 
+    only returned `ReadOnlyDatabase`. Therefore, I had to find a workaround for that.
+  * Implemented `Leave` class
+    * Created the `Leave` class one of the key components we are tracking in our application
+    * Use Case: Building blocks to track stuff like leaves taken so far, whether employee is on leave etc.
+    * Why: Tracking Leaves is one of the more important components of HRM
+    * Constraints: Was not too difficult other than the part about writing into `JSON Adapted Person` 
+     which was slightly more complicated as I had to make sure that the Priority Queue of the `Leave` will 
+    be stored in the `database.json`.
```
Comment (chao890) [inline docs/team/prit3010.md:38]: Can standardize punctuations here

**Review by chao890 — CHANGES_REQUESTED** [review]

**Review by Kok-je — APPROVED** [review]

**Review by chao890 — APPROVED** [review]

**Pull request merged**